packages/grafana-ui/src/components/DataLinks/FieldLink.tsx

```diff
@@ -0,0 +1,64 @@
+import { Field, GrafanaTheme, LinkModel } from '@grafana/data';
+import { css } from 'emotion';
+import React from 'react';
+import { useStyles } from '../../themes';
+import { Icon } from '../Icon/Icon';
+import { DataLinkButton } from './DataLinkButton';
+
+type Props = {
+  links: Array<LinkModel<Field>>;
+};
+
+export function FieldLink({ links }: Props) {
```

Comment:
Should this be a FieldLinkList or something like that?

Reply:
yea, this looks like a list

Reply:
fixed